src/mongocxx/test/spec/unified_tests/runner.cpp

```diff
@@ -73,6 +74,88 @@ entity::map& get_entity_map() {
     return init_maps().second;
 }
 
+const auto kLocalMasterKey =
+    "\x32\x78\x34\x34\x2b\x78\x64\x75\x54\x61\x42\x42\x6b\x59\x31\x36\x45\x72"
+    "\x35\x44\x75\x41\x44\x61\x67\x68\x76\x53\x34\x76\x77\x64\x6b\x67\x38\x74"
+    "\x70\x50\x70\x33\x74\x7a\x36\x67\x56\x30\x31\x41\x31\x43\x77\x62\x44\x39"
+    "\x69\x74\x51\x32\x48\x46\x44\x67\x50\x57\x4f\x70\x38\x65\x4d\x61\x43\x31"
+    "\x4f\x69\x37\x36\x36\x4a\x7a\x58\x5a\x42\x64\x42\x64\x62\x64\x4d\x75\x72"
+    "\x64\x6f\x6e\x4a\x31\x64";
+
+bsoncxx::document::value get_kms_values() {
+    char key_storage[96];
+    memcpy(&(key_storage[0]), kLocalMasterKey, 96);
+    const bsoncxx::types::b_binary local_master_key{
+        bsoncxx::binary_sub_type::k_binary, 96, (const uint8_t*)&key_storage};
+
+    const auto kms_doc = make_document(
```

Comment:
```suggestion
    auto kms_doc = make_document(
```
Do not prevent NRVO.